Error when trying to install blocktest

I was trying to install this repo following the instructions in the README.md but I retrieved the following error:

```shell
/home/mgloria/iit/blocktest/build/_deps/unittest-src/googletest/src/gtest-death-test.cc: In function ‘bool testing::internal::StackGrowsDown()’:
/home/mgloria/iit/blocktest/build/_deps/unittest-src/googletest/src/gtest-death-test.cc:1301:24: error: ‘dummy’ may be used uninitialized [-Werror=maybe-uninitialized]
 1301 |   StackLowerThanAddress(&dummy, &result);
      |   ~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
/home/mgloria/iit/blocktest/build/_deps/unittest-src/googletest/src/gtest-death-test.cc:1290:13: note: by argument 1 of type ‘const void*’ to ‘void testing::internal::StackLowerThanAddress(const void*, bool*)’ declared here
 1290 | static void StackLowerThanAddress(const void* ptr, bool* result) {
      |             ^~~~~~~~~~~~~~~~~~~~~
/home/mgloria/iit/blocktest/build/_deps/unittest-src/googletest/src/gtest-death-test.cc:1299:7: note: ‘dummy’ declared here
 1299 |   int dummy;
      |       ^~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [_deps/unittest-build/googletest/CMakeFiles/gtest.dir/build.make:90: _deps/unittest-build/googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:462: _deps/unittest-build/googletest/CMakeFiles/gtest.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

This seems an error related to gtest, not this repo. Looking at the gtest issues, I found that a lot of people have experienced this bug and this happens with `gcc version 11.x` and `googletest v1.10` (which I think is the version used in here). 
This issue got fixed by [this PR](https://github.com/google/googletest/pull/3024) and the fix is available since gtest `v1.11.0`. 

To correctly install BlockTest, maybe it is sufficient to update the [`GIT_TAG`](https://github.com/robotology/blocktest/blob/2a9acc383d0533492cc3986a4fc328f5dc4bb4a4/src/CMakeLists.txt#L22) argument of the `FetchContent_Declare` call.

--- a/README.md
+++ b/README.md
@@ -1,13 +1,11 @@
 # 1. Maintainers
 
 
-| | CI and deployment |
+| | |
 |:---:|:---:|
 | [<img src="https://github.com/Nicogene.png" width="40">](https://github.com/niNicogenecogene) | [@Nicogene](https://github.com/Nicogene) |
-
-| | Development |
-|:---:|:---:|
 | [<img src="https://github.com/triccyx.png" width="40">](https://github.com/triccyx) | [@triccyx](https://github.com/triccyx) |
+
 
 # 2. CI Status
 

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -19,7 +19,7 @@ if(COMPILE_WITHUNITTEST AND Boost_VERSION_STRING GREATER_EQUAL 1.66.0)
   FetchContent_Declare(
     unittest
     GIT_REPOSITORY https://github.com/google/googletest.git
-    GIT_TAG        release-1.10.0
+    GIT_TAG        release-1.11.0
   )
   FetchContent_GetProperties(unittest)
   if(NOT unittest)

